app/assets/javascripts/mumuki_laboratory/application/interval.js

```diff
@@ -1,6 +1,4 @@
-var mumuki = mumuki || {};
-
-(function (mumuki) {
+(function () {
```

Comment:
Prefer ES lambda syntax for consistency :blush:

Reply:
I dind't to change this one since it uses `this` in a way I am not 100% sure it will work